Makefile

```diff
@@ -4,3 +4,28 @@ include ./static-commontooling/make/lib_static_commontooling.mk
 include ./static-commontooling/make/standalone.mk
 include ./static-commontooling/make/pythonic.mk
 include ./static-commontooling/make/docker.mk
+
+all: all
```

Comment:
We tend to define help docs as their own target and add that as a pre-requisite to all. This is so all of the help docs in all relevant makefiles are outputted. This will work as this is the only place you're defined a concrete recipe for the target. But if it someone else decided to do the same elsewhere in future, it wouldn't work as expected. It's probably best to stick to the same pattern here for consistency and to avoid others in future copy-pasting a pattern that might not work in other contexts. I'm also unsure why this target has itself as a pre-requisite?